assets/pages/new-contributions-landing/components/ButtonWithRightArrow.jsx

```diff
@@ -20,7 +20,7 @@ function ButtonWithRightArrow(props: PropTypes) {
 
   return (
     <div className={props.componentClassName}>
-      <button
+      <a
```

Comment:
could this have an `aria-role="button"`? Provided it's being used as a button. If it's normally used as a link then it's actually better now!